pkgs/development/python-modules/pushover/default.nix

```diff
@@ -0,0 +1,24 @@
+{ stdenv, buildPythonPackage, fetchPypi
+, requests }:
+
+buildPythonPackage rec {
+  pname = "python-pushover";
+  version = "0.3";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "0xlghiqd9rsgn7jdhc8v1xh3xspssihrw1vyy85gvjzxa1ah19sk";
+  };
+
+  propagatedBuildInputs = [ requests ];
+
+  # checks crash
+  doCheck = false;
```

Comment:
Tests could be run using `python test.py` but they require network access: https://github.com/laprice/pushover/blob/master/test.py